core/src/main/java/bisq/core/payment/PaymentAccount.java

```diff
@@ -148,6 +149,20 @@ public TradeCurrency getSingleTradeCurrency() {
             return null;
     }
 
+    @Override
```

Comment:
This seems to work even though the EqualsAndHashCode annotation is still on the class. But, if you are overriding them it may make sense to remove it so lombok doesn't generate the code or future readers assume the generate equals is valid for this class.